hermes/bin/Cargo.toml

```diff
@@ -41,8 +41,17 @@ bip32 = { workspace = true }
 ed25519-bip32 = { workspace = true }
 dashmap = { workspace = true }
 once_cell = { workspace = true }
+clap = { workspace = true, features = ["derive", "env"] }
+tracing = { workspace = true, features = ["log"] }
+tracing-subscriber = { workspace = true, features = ["fmt", "json", "time"] }
+build-info = { workspace = true }
+derive_more = { workspace = true }
```

Comment:
It would be neat if you specify the feature you are using. In this case should be only `display` with `default-features = false`